Example/src/SwipeableListExample.tsx

```diff
@@ -49,8 +55,9 @@ const data = [
   },
 ];
 
-export default function SwipableList() {
+const SwipableList: FC = () => {
   function onRemove() {
+    // @ts-ignore: FIXME shouldn't be Alert.alert()?
```

Comment:
I changed all `alert` to `Alert.alert` in RNGH examples iirc.

Reply:
So I'll fix that one as well.

Reply:
Fixed in 5e175d21.